nix/test/basic/fio_nvme_basic.nix

```diff
@@ -0,0 +1,109 @@
+{ pkgs, lib, ... }:
+let
+  targetIp = "192.168.0.1";
```

Comment:
Doesn't a fixed ip address limit where this test can be deployed?

Reply:
No, the network is a VLAN -- and is isolated from the host. You can not reach this network with your own host/machine.